libs/ledger-live-common/src/exchange/providers/sell.ts

```diff
@@ -31,19 +31,15 @@ export const fetchAndMergeProviderData = async () => {
     return providerDataCache;
   }
   try {
-    const [sellProvidersData, fundProviderData] = await Promise.all([
-      getProvidersData("sell"),
-      getProvidersData("fund"), // Mercuryo is currently treated as a fund provider
-    ]);
-    providerDataCache = { ...sellProvidersData, ...fundProviderData };
-    return providerDataCache;
+    const sellProvidersData = await getProvidersData("sell");
+    return { ...sellProvidersData };
```

Comment:
The cache was introduced because old swaps spammed this network call. 
With Live apps I can understand you want to remove this. But if you stop using just remove all of its reference in the file

Reply:
Happy to remove it!